### ISSUE TITLE
Flipped screen on 1.16+ using update on fixed black screen 1.13+

### ISSUE THREAD

**Issue by kedrick021** [PojavLauncherTeam/PojavLauncher#2465]
### Describe the bug

Bbn

### The log file and images/videos

Jjj

### Steps To Reproduce

```markdown
Kk
```

### Expected Behavior

Jj

### Platform

```markdown
- Device model: 
- CPU architecture: 
- Android version: 
- PojavLauncher version:
```

**Comment by ghost** [comment]
Wtf

**Comment by GoneSubject1** [comment]
._.

**Comment by GoneSubject** [comment]
._.

**Comment by NdraaGaminz** [comment]
Download PojavLauncher Fixed Flipped Screen on Action menu

**Comment by GoneSubject** [comment]
> Download PojavLauncher Fixed Flipped Screen on Action menu

so it fixed right?

**Comment by Fl414r** [comment]
The joke here is that Fixed Black Screen 1.13+ version comes AFTER Fixed Flipped Screen version. The developer repaired the screen and broke it immediately.

**Comment by kedrick021** [comment]
I mean fixed black screen on 1.13+ when I change the render 2 on 1.16+ is flipped screen

And latest update too
